### DISCUSSION TITLE
add Hyphens into word Break

### DISCUSSION THREAD

**Discussion by geraldofebrian** [tailwindlabs/tailwindcss#8508] [Ideas]
https://tailwindcss.com/docs/word-break

i have a request to add `hyphens: auto`. This can give the `-` when the words break. Thus the generated css would be:

```
.break-all {
    word-break: break-all;
    hyphens: auto;
}
.break-words {
    overflow-wrap: break-word;
    hyphens: auto;
}
.break-normal {
    overflow-wrap: normal;
    word-break: normal;
    hyphens: auto;
}
```

But, i don't know how to customize the utlities. any help? thank you.

**Comment by fratzinger** [comment]
I've created a PR: https://github.com/tailwindlabs/tailwindcss/pull/10071
